hw/top_earlgrey/cdc/cdc_waivers.misc.tcl

```diff
@@ -4,76 +4,99 @@
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_upload.u_arbiter.u_req_fifo.gen_normal_fifo.u_fifo_cnt.rptr_o[2:0]") && (Signal == "top_earlgrey.u_spi_device.u_memory_2p.b_rvalid_sram_q") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.readbuf_addr_busclk[31:0]") && (Signal == "top_earlgrey.u_spi_device.u_readcmd.readbuf_addr[31:0]") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_rv_dm.dap.i_dmi_jtag_tap.jtag_ir_shift_q[4]") && (Signal == "IOR2") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_intr_state_generic_rx_full.q[0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[11:2]") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_intr_state_generic_rx_watermark.q[0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[11:0]") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_rv_dm.dap.i_dmi_jtag_tap.tap_state_q[3:0]") && (Signal == "IOR0") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_rxf_ptr_wptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[12:0]") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
-set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_txf_ptr_rptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_txf_ctrl.rptr[12:0]") && (Association == "None")} -status {Waived} -lastedit_user {root} -lastedit_time {Monday, 14 November 2022 20:21:17 PDT}
+# W_DATA : included in waived paths
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_pinmux_aon.mio_oe_retreg_q[46:0]") && (Signal == "top_earlgrey.u_spi_host1.u_cmd_queue.cmd_fifo.gen_normal_fifo.u_fifo_cnt.rptr_o[2:0]") && (Association == "None")} -status {Waived} -comment {included in waived paths}
```

Comment:
for this first batch of waivers regarding the retreg, what has changed? It looks like most of them are identical... is it only the comment that has changed?

if so, it would be good to split these into a separate commit so that it is clear that these are not "new" waivers per se, but just waivers with updated comments (this will make the review quite a bit easier).

Reply:
Yes, only difference is the comment. cdc_waivers.misc.tcl includes additional paths associated with waived paths. I will split this file into a separate commit.

Reply:
ok - it would also be good if you could split the reformatting of existing waivers and the addition of new waivers into different commits. let me know if you need tips for how to do that with GIT.

Reply:
I split it and added cdc_waivers.misc.tcl into another commit. Let me know if it looks OK.